packages/flutter/test/material/outlined_button_test.dart

```diff
@@ -2,6 +2,11 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// This file is run as part of a reduced test set in CI on Mac and Windows
+// machines.
+@Tags(<String>['reduced-test-set'])
```

Comment:
Just a question. Why do we use this tag here? Not sure the usage of this tag:)

Reply:
This is required for test class with golden tests.
https://github.com/flutter/flutter/wiki/Writing-a-golden-file-test-for-package%3Aflutter#reduced-test-set-tag

Since M3 Ink well shaders can't be tested from mock canvas. We discussed and agreed to swap it for a golden test. 
https://github.com/flutter/flutter/pull/136809#discussion_r1365053630